src/constants/locales.ts

```diff
@@ -33,7 +33,7 @@ export const SUPPORTED_LOCALES = [
   'zh-CN',
   'zh-TW',
 ] as const
-export type SupportedLocale = typeof SUPPORTED_LOCALES[number] | 'pseudo'
+export type SupportedLocale = typeof SUPPORTED_LOCALES[number] | 'pseudo' | string
```

Comment:
This should _not_ include `string` - it's specifically a list of our supported types.

Reply:
unfortunately it has to in order to support usage w/ lingui's locale hook